MetaMask/metamask-extension
# Disable Previous Button on First Page

## Explanation

Previous Page button was enabled for hardware wallets despite being on the first page.

### After

![image](https://user-images.githubusercontent.com/36251325/216845269-25a7d6c1-4275-46e8-8279-ab62d2b216ba.png)

## Manual Testing Steps

- Connect Ledger
- Previous Page button should be initially disabled
- Go a page forward and then back
- Repeat with Lattice, Trezor etc.

## Pre-merge author checklist


## Pre-merge reviewer checklist
